### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Delete_DeletePopup_Screenreader : screen reader Jaws is announcing the data from background when "Delete" dialog opens..

### ISSUE THREAD

**Issue by prasadaps** [microsoft/AzureStorageExplorer#3450]
## Bug Description ##
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#JAWS;#Win32;#WCAG1.3.1;#A11ySev2;#Benchmark;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version:  1.14.2
Windows Version - Windows10 
Screen reader: JAWS(2020.1912.11ILM), NVDA Version:2020.2

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Once Completed, Right click on uploaded blob and Select "Delete" and hit enter.
7. Start screen reader Jaws and observe when Delete dialog opens the screen reader announce table data from background.

### Actual:
When delete dialog opens the screen reader announce the table data from background.

**Note** :  This issue is only observed with the screen reader Jaws and not with screen reader NVDA.

### Expected:
The screen reader should announce the data available on Delete dialog only when its open.

### User Impact: 
The screen reader users my confused about the data they may think the dialog is also contain the data announce by screen reader which is actually present in background and not on delete dialog.

### Attachment
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-prphat_microsoft_com/EaJ-sPqaCtxFlItGftCi2qYB1nv-ggOV2UixmgkBanKMdA?e=ldBHBD

**Comment by craxal** [comment]
@prasadaps This appears to be a known issue with JAWS. See FreedomScientific/VFO-standards-support#160.

**Comment by craxal** [comment]
I can no longer reproduce. I believe this is due to the fix for #3479. The table no longer has an assertive live region.

**Comment by prasadaps** [comment]
Verified the issue in latest version of Azure tool storage explorer application the issue is no longer reproducing so closing the bug.
please refer the closing attachment. 
https://microsoft-my.sharepoint.com/:v:/p/v-rojad/EXfIy1VtNNdLnFP7eu9liSQBPETiZVi_oatRbbs01kr8hA?e=SEACLK

**Comment by 12-shweta** [comment]
**Github Tags**:#Closed;#Regressed:02-03-23;
